sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj

```diff
@@ -20,4 +20,8 @@
     <AzureApiTag />
     <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
   </PropertyGroup>
+
+  <PropertyGroup >
+    <DocumentationFile>$(IntermediateOutputPath)$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFile>
```

Comment:
@chidozieononiwu is this needed or we have some other way to pull this in?

Reply:
Yes it is required. Although it is being pulled in as part of the build [here](https://github.com/Azure/azure-sdk-for-net/blob/4c390c8bdb5fe020e7890a7d5129e7222db965fc/eng/Directory.Build.Data.props#L118). Right now it is conditioned for only Track 2 libraries. Track 1 libraries have been adding the DocumentationFile seperately.